DigitalLearningSolutions.Web.Tests/Helpers/FilteringHelperTests.cs

```diff
@@ -93,14 +94,17 @@ public void AddNewFilterToFilterBy_doesnt_append_with_null_new_filter()
             result.Should().Be("Test");
         }
 
-        [Test]
-        public void AddNewFilterToFilterBy_doesnt_append_with_new_filter_already_in_filterBy()
+        [TestCase("Test", "Test")]
+        [TestCase(FilterByAlphaBravoCharlie, "Group|Name|Alpha")]
+        [TestCase(FilterByAlphaBravoCharlie, "Group|Name|Bravo")]
+        [TestCase(FilterByAlphaBravoCharlie, "Group|Name|Charlie")]
+        public void AddNewFilterToFilterBy_doesnt_append_with_new_filter_already_in_filterBy(string filterBy, string newFilterValue)

```

Comment:
Do we need all these test cases/what different edge cases do these cover + same question about the test below?

Reply:
They are in different parts of the existing filter string (beginning, middle and end), and different substrings of an existing filter in the filter string (beginning, middle and end). Just makes sure it splits the filters and checks properly; e.g. an implementation which just uses includes() might miss a case. Not really needed since its a very simple function so I've removed some.